etcdserver/raft.go

```diff
@@ -223,6 +226,8 @@ func (r *raftNode) start(rh *raftReadyHandler) {
 				// gofail: var raftAfterSave struct{}
 
 				if !raft.IsEmptySnap(rd.Snapshot) {
+					// waits etcd server to finish renaming snap db to db.
+					<-replaceDBDone
```

Comment:
kk, it seems to me the raftDone should act as execution flow chan between etcd server and raft since both need to send msg to it. I am not sure if the name `raftDone` make sense anymore.